modules/nw-enforcing-hsts-per-domain.adoc

```diff
@@ -25,13 +26,17 @@ HSTS cannot be applied to insecure, or non-TLS routes, even if HSTS is requested
 ====
 
 .Prerequisites
-
+ifndef::microshift[]
 * You are logged in to the cluster with a user with administrator privileges for the project.
-* You installed the `oc` CLI.
+endif::microshift[]
+ifdef::microshift[]
+* You have root access to the cluster.
+endif::microshift[]
+* You installed the {oc-first}.
 
 .Procedure
 
-. Edit the Ingress config file:
+. Edit the Ingress configuration YAML by running the following command and updating fields as needed:
```

Comment:
Should this be within `ifndef` sections?

Reply:
This is a one-step procedure in the OCP docs. We can remove this step, but that leaves just step 2, which is using an annotation: "You can apply HSTS to all routes in the cluster or in a particular namespace by entering the oc annotate command." Do the rest of the fields in the Ingress object not apply?

Reply:
Wait a minute, I overlooked that this was `Ingress` from `config.openshift.io` apigroup, not kubernetes. This whole part may not apply then, sorry for the confusion. The part about configuring hsts in each route through is still possible though.

Reply:
Updated with a microshift-only module. See what you think? Starts here in the diff https://github.com/openshift/openshift-docs/pull/76762/files#diff-c0ceafbc8ba7038fbab4badf0effbb0285791ddd39a93cdb199d7a85eb1427e9R1 same title in the doc preview.